www/src/examples/ModalStatic.js

```diff
@@ -1,14 +1,18 @@
-<div className="static-modal">
-  <Modal.Dialog>
-    <Modal.Header>
-      <Modal.Title>Modal title</Modal.Title>
-    </Modal.Header>
+const modalInstance = (
```

Comment:
this looks like a bad merge, the old code is more correct (unnecessary render())

Reply:
Fixed